.github/workflows/bundle_with_dakota.yml

```diff
@@ -0,0 +1,161 @@
+name: 🏎️ Make & Test Wheels 🏎️
+
+on: [pull_request]
+
+env:
+  ERT_SHOW_BACKTRACE: 1
+  NO_PROJECT_RES: 1
+  BOOST_VERSION: 1.82.0
+  BOOST_VERSION_UNDERSCORES: 1_82_0
+  DAKOTA_VERSION: 6.18.0
+  INSTALL_DIR: local
+  FORCE_REBUILD: false
+
+jobs:
+  build_wheels:
+    name: 🛞 Build Wheels 🛞
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.8','3.9','3.10'] # 3.11 not needed at the moment
```

Comment:
Update comment because Everest prohibits testing due to not supporting anything above 3.10 atm.